src/languages/es.js

```diff
@@ -878,6 +878,8 @@ export default {
         clearStatus: 'Borrar estado',
         save: 'Guardar',
         message: 'Mensaje',
+        untilTomorrow: 'Hasta mañana',
+        untilTime: ({time}) => `Hasta las ${time}`,
```

Comment:
@stitesExpensify Can you confirm translations are good internally?

Reply:
Unfortunately we need to change this 😞 

When the time starts with `1` we need it to be `Hasta la`

Reply:
No es una problema. Yo también. 

That being said, I think I was unclear. It needs to be:

`01:xx` -> Hasta la
Any time that does not start with `01` including `00` -> Hasta las

Reply:
What about a format with days like `Until 06-29 11:30AM`. Should we use `Hasta el día`?

Reply:
Thanks for the details!
What about a format with days like `Until 06-29 11:30AM`. Should we use `Hasta el día`?